diff --git a/frontend/src/components/Student.js b/frontend/src/components/Student.js
index e329873..bd8f01a 100644
--- a/frontend/src/components/Student.js
+++ b/frontend/src/components/Student.js
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import TextField from '@material-ui/core/TextField';
 import { Container ,Paper,Button} from '@material-ui/core';
+const PUBLIC_URL = process.env.REACT_APP_PUBLIC_URL || 'localhost'
 
 const useStyles = makeStyles((theme) => ({
   root: {
@@ -23,7 +24,7 @@ export default function Student() {
     e.preventDefault()
     const student={name,address}
     console.log(student)
-    fetch("http://localhost:8080/student/add",{
+    fetch(PUBLIC_URL + '/student/add',{
       method:"POST",
       headers:{"Content-Type":"application/json"},
       body:JSON.stringify(student)
@@ -34,7 +35,7 @@ export default function Student() {
 }
 
 useEffect(()=>{
-  fetch("http://localhost:8080/student/getAll")
+  fetch(PUBLIC_URL + '/student/getAll')
   .then(res=>res.json())
   .then((result)=>{
     setStudents(result);
